source/mainnet/docs/smart-contracts/guides/build-schema.rst

```diff
@@ -159,19 +159,6 @@ have the options to embed the schema and/or write the schema to a file and/or pr
    For more on which to choose see
    :ref:`here<contract-schema-which-to-choose>`.
 
```

Comment:
As this section is about how to embed a schema or write it to I file, I would keep a section explain the embedded schema. Something along the line:

Embedding the schema
-----------------------------------

The schema is embedded by default when building the smart contract module:
```
cargo concordium build 
```

If successful the output of the command will tell you the total size of the
schema in bytes.

You can disable the default behavior (not recommended) with:

```
cargo concordium build --no-schema-embed
```